src/components/artwork/details.tsx

```diff
@@ -19,7 +19,7 @@ export class ArtworkDetails extends React.Component<DetailsProps, null> {
     } else if (artists && artists.length) {
       let artistsEl = []
       for (let i = 0; i < artists.length; i++) {
-        artistsEl.push(<TextLink href={ artists[i].href }>{ artists[i].name }</TextLink>)
+        artistsEl.push(<TextLink href={ artists[i].href } key={ i }>{ artists[i].name }</TextLink>)
```

Comment:
👍  I just sent a new pull request that addresses the key issue: https://github.com/artsy/reaction-force/pull/36/files